### PR TITLE
[2.3] New Docker Compose environment for developers

### PR DESCRIPTION
**Pull request by sevein**
See: https://www.accesstomemory.org/en/docs/2.3/dev-manual/env/compose/

It targets development. I don't think we are ready for production yet.

You'll likely see grammar mistakes, please let me know and forgive my English!

### CODE REVIEW

File: dev-manual/env/compose.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+   .. code-block:: bash
+
+      docker-compose exec atom curl elasticsearch:9200/_cat/nodes
+
+      cdec404af617 172.18.0.7 6 65 0.26 d m Huntara
+      366a7817864f 172.18.0.3 5 65 0.26 d * Vance Astro
+      0e52024208fe 172.18.0.6 4 65 0.26 d m Bloodstorm
+
+You can also verify that two workers has subscribed to Gearman:
```
Comment (fiver-watson) [inline dev-manual/env/compose.rst]: two workers *_have *_ subscribed to Gearman

File: dev-manual/env/compose.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   .. code-block:: bash
+
+      docker-compose exec atom bash -c "echo STATUS | nc gearmand 4730"
+
+      fdd4764376d2f763-arGenerateFindingAidJob            0   0   2
+      fdd4764376d2f763-arUpdatePublicationStatusJob       0   0   2
+      fdd4764376d2f763-arInformationObjectCsvExportJob    0   0   2
+      fdd4764376d2f763-arInheritRightsJob                 0   0   2
+
+You could temporary stop all the services or both stop and remove related
```
Comment (fiver-watson) [inline dev-manual/env/compose.rst]: temporarily

**Comment by fiver-watson** [comment]
Looks great! I made a couple minor comments but otherwise it's perfect :fireworks:

**Comment by sevein** [comment]
thanks thanks

On Wed, Apr 27, 2016 at 1:21 PM, Dan Gillean notifications@github.com
wrote:

> Looks great! I made a couple minor comments but otherwise it's perfect [image:
> :fireworks:]
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/artefactual/atom-docs/pull/29#issuecomment-215215935

## 

Jesús García Crespo,
Software Engineer, Artefactual Systems Inc.
http://www.artefactual.com | +1.604.527.2056

**Pull request merged**